src/pages/ServiceDetails/ServiceInfo/ServiceInfoDescription.tsx

```diff
@@ -67,106 +71,115 @@ class ServiceInfoDescription extends React.Component<ServiceInfoDescriptionProps
-                            >
-                              <strong>{address.ip} </strong>
-                            </OverlayTrigger>
-                          </li>
-                        );
-                      })}
-                    </ul>
-                  </Col>
-                </Row>
-              ))}
-            </Col>
-            <Col xs={12} sm={6} md={3} lg={3}>
-              <div className="progress-description">
-                <strong>Health</strong>
-              </div>
-              <HealthIndicator
-                id={this.props.name}
-                health={this.props.health}
-                mode={DisplayMode.LARGE}
-                tooltipPlacement="left"
-              />
-            </Col>
-          </Row>
-        </div>
-      </div>
+                </StackItem>
+                <StackItem id={'created_at'}>
+                  <Text component={TextVariants.h3}> Created at </Text>
+                  <LocalTime time={this.props.createdAt} />
+                </StackItem>
+              </Stack>
+            </CardBody>
+          </Card>
+        </GridItem>
+        <GridItem span={6}>
+          <Card style={{ height: '100%' }}>
+            <CardBody>
+              <Title headingLevel="h3" size="2xl">
+                {' '}
+                Health Overview{' '}
+              </Title>
+              <Stack>
+                <StackItem id={'health'}>
+                  <Text component={TextVariants.h3}> Health </Text>
+                  <HealthIndicator
+                    id={this.props.name}
+                    health={this.props.health}
+                    mode={DisplayMode.LARGE}
+                    tooltipPlacement="left"
+                  />
+                </StackItem>
+              </Stack>
+              <Title headingLevel="h3" size="2xl" style={{ marginTop: '60px' }}>
+                {' '}
+                Network Overview{' '}
+              </Title>
+              <Stack>
+                <StackItem id={'ip'}>
+                  <Text component={TextVariants.h3}>
+                    {' '}
+                    {this.props.type !== ExternalNameType ? 'Service IP' : 'ExternalName'}{' '}
+                  </Text>
+                  {this.props.type !== ExternalNameType
+                    ? this.props.ip
+                      ? this.props.ip
+                      : ''
+                    : this.props.externalName
+                    ? this.props.externalName
+                    : ''}
+                </StackItem>
+                <StackItem id={'endpoints'}>
+                  <Text component={TextVariants.h3}> Endpoints </Text>
+                  <Stack>
+                    {(this.props.endpoints || []).map((endpoint, i) =>
+                      (endpoint.addresses || []).map((address, u) => (
+                        <StackItem key={'endpoint_' + i + '_address_' + u}>
+                          <Tooltip content={<>{address.name}</>}>
```

Comment:
I think that address can be empty. Shouldn't Kiali prevent the tooltip and the eye when there name is empty?

![Screenshot of Kiali Console (32)](https://user-images.githubusercontent.com/613814/65527494-07c0b600-def3-11e9-903c-7194ec55ee0a.jpg)